DESCRIPTION

```diff
@@ -26,7 +26,8 @@ Imports:
     snakecase (>= 0.9.2),
     magrittr,
     purrr,
-    rlang
+    rlang,
+    tidyselect
```

Comment:
let's specify at least version 1.0.0 now that it's on CRAN

Reply:
done.  And thanks for the invite to add myself as contrib!